modules/update-preparing-forced-update-risk.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_content-type: CONCEPT
+[id="update-preparing-forced-update-risk"]
+= Assessing the risk of forced updates
+
+If there is no other way left to update a cluster, you can "force" an update by using the `force` option in the CLI. It's important to understand the risks of performing a forced update to determine if this decision outweighs the potential risks. 
```

Comment:
IMO we should replace `force` with `--force`. Also for awareness here is text from OC

```
$ oc adm upgrade --help

 The cluster may report that the upgrade should not be performed due to a content verification error or update
precondition failures such as operators blocking upgrades. Do not upgrade to images that are not appropriately signed
without understanding the risks of upgrading your cluster to untrusted code. If you must override this protection use
the --force flag.

--force=false:
	Forcefully upgrade the cluster even when upgrade release image validation fails and the cluster is reporting
	errors.
```

Reply:
Sounds good! Done :)